modelling/src/test/java/org/axonframework/modelling/SimpleStateManagerTest.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (c) 2010-2025. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.modelling;
+
+import org.axonframework.messaging.StubProcessingContext;
+import org.axonframework.messaging.unitofwork.ProcessingContext;
+import org.axonframework.modelling.repository.AsyncRepository;
+import org.junit.jupiter.api.*;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class SimpleStateManagerTest {
+
+    private final AsyncRepository<String, Integer> repository = new SimpleEntityAsyncRepository<>(
+            String.class,
+            Integer.class,
+            (id, context) -> CompletableFuture.completedFuture(Integer.parseInt(id)),
+            (id, entity, context) -> CompletableFuture.completedFuture(null)
+    );
+
+    @Test
+    void registerAllowsEntitiesToBeLoadedFromTheRepository() {
+        // Given
```

Comment:
Very, very small think - not important, I just use lower case versions of those, more looking like "java", in C# I use "UpperCase", but up to you or better we can decide to use one convention for the comments. 
Choose what is better for you, I can do uppercase if following tasks :)

Reply:
I am not really biased either way. Adjusting it to lowercase to align with your changes.
I see a mix of both throughout the framework actually